include/envoy/server/instance.h

```diff
@@ -213,6 +206,11 @@ class Instance {
    * @return the flush interval of stats sinks.
    */
   virtual std::chrono::milliseconds statsFlushInterval() const PURE;
+
+  /**
```

Comment:
Why is this still here? All of this should be gone from the public interfaces.

Reply:
This is because of the second singleton problem I am talking about. If there is a way to get singleton name/it is ok to hard code the singleton name, I can remove this.

Reply:
I have removed from this and other public interfaces.